packages/jest-types/src/Global.ts

```diff
@@ -59,7 +59,7 @@ interface Each<EachFn extends TestFn | BlockFn> {
   // when the table is an array of object literals
   <T extends Record<string, unknown>>(table: ReadonlyArray<T>): (
     name: string | NameLike,
-    fn: (arg: T) => ReturnType<EachFn>,
+    fn: (arg: T, done: DoneFn) => ReturnType<EachFn>,
```

Comment:
not sure this is correct. We only pass `done` if the test function takes more arguments that there are entries in the provided test data. See this `length` check:

https://github.com/facebook/jest/blob/2993a4f4845c0bebafb7cac95ae751615272f16c/packages/jest-each/src/bind.ts#L106

So `done` is not always provided. Unsure if it's possible to say that with types? I guess instead of arrays we'd need to use tuples and then somehow count the arguments as well?

Reply:
Interesting. I missed that line. Will try to see if counting arguments could work in types.